### ISSUE TITLE
Side by side view bigger than screen

### ISSUE THREAD

**Issue by sirnacnud** [ForkIssues/Tracker#161]
When using the side by side view to view the diff of a change, Fork creates a view bigger than the laptop screen (both wider and taller). I am only able to scroll the view up and down, I can't scroll it back and forth. The close button is off screen so I'm not able to close it out.

**Comment by DanPristupov** [comment]
@sirnacnud that is right. Fork remembers the last window position and size and restores them next time. I need to also check if the window fits into the _current_ screen, otherwise contract it.

Going to fix this in the next update.
